packages/e2e-tests/specs/editor/various/rtl.test.js

```diff
@@ -15,10 +15,12 @@ const ARABIC_TWO = '٢';
 describe( 'RTL', () => {
 	beforeEach( async () => {
 		await createNewPost();
+		await page.evaluate( () => {
+			document.querySelector( '.is-root-container' ).dir = 'rtl';
```

Comment:
So it seems reducing the scope of the layout shift is solving the issue. We're just testing rich text and writing flow here, so that's fine.

Reply:
That is weird but I guess it's better than having wrong snapshots :)